filters-share/adservice_firstparty.txt

```diff
@@ -34,7 +34,6 @@
 ||ad-img.ebaykorea.com^
 ||api.ezadsystem.com^
 ||ad.planbplus.co.kr^
-||api.linkmine.co.kr^
```

Comment:
[Its website](http://linkmine.co.kr/) exists.

Reply:
The entry still exists at https://github.com/iam-py-test/List-KR/blob/master/filters-share/adservice_firstparty.txt#L31 (I should have clarified that in my original PR, sorry)